ui/src/pages/server-types/server-types.tsx

```diff
@@ -50,9 +50,18 @@ export const ServerTypes = (): React.ReactElement => {
     data: serverTypes,
     isLoading,
     error,
-  } = useQuery(['serverTypes'], async () => {
-    const { data } = await axios.get('/spawner-profiles/');
-    return data;
+  } = useQuery<AppProfileProps[], { message: string }>({
+    queryKey: ['server-types'],
```

Comment:
I see, you changed it here! Is this related to upgrade of React Query?

Reply:
Yup, they changed how you have to use the hook